### PR TITLE
Environment_Engine: Intuitive default environmental panel types

### LINKED ISSUE #2795: Default environment panel types
#### Description:

The default panel type assignment for from to spaces isn't intuitive. 
image

Can we change it so that:
SlabOnGrade to Floor
WallExternal to Wall
Roof to Ceiling

Thanks!

### PR DESCRIPTION
**Pull request by albinber**
### Issues addressed by this PR

Closes #2795

### CODE REVIEW

**Comment by albinber** [comment]
@bhombot check compliance

File: Environment_Engine/Create/Panel.cs [hunk 1]
```diff
@@ -49,15 +49,15 @@ public static partial class Create
         [Input("angleTolerance", "The angle tolerance for collapsing to polylines used when generating the external edges of the surfaces")]
         [Input("panelType", "The panel type to assign to the panels, if no input is added roof, floor and wall types are assigned by default")]
         [Output("panelsAsSpace", "A collection of Environment Panels representing a closed space generated from the provided Brep geometry")]
-        public static List<Panel> Panels(this BoundaryRepresentation brep, string connectedSpaceName = null, double angleTolerance = BH.oM.Geometry.Tolerance.Angle, PanelType panelType = PanelType.Undefined)
+        public static List<Panel> Panels(this BoundaryRepresentation brep, string connectedSpaceName = null, double angleTolerance = BH.oM.Geometry.Tolerance.Angle, PanelType roofType = PanelType.Roof, PanelType ceilingType = PanelType.Ceiling, PanelType internalFloorType = PanelType.FloorInternal, PanelType externalGrounding = PanelType.SlabOnGrade, PanelType externalWallType = PanelType.WallExternal, PanelType internalWallType = PanelType.WallInternal)
```
Comment (FraserGreenroyd) [inline Environment_Engine/Create/Panel.cs:52]:
```suggestion
        [PreviousVersion("6.3", "BH.Engine.Environment.Create.Panels(BH.oM.Geometry.BoundaryRepresentation, System.String, System.Double, BH.oM.Environment.Elements.PanelType)")]
        public static List<Panel> Panels(this BoundaryRepresentation brep, string connectedSpaceName = null, double angleTolerance = BH.oM.Geometry.Tolerance.Angle, PanelType roofType = PanelType.Roof, PanelType ceilingType = PanelType.Ceiling, PanelType internalFloorType = PanelType.FloorInternal, PanelType externalGrounding = PanelType.SlabOnGrade, PanelType externalWallType = PanelType.WallExternal, PanelType internalWallType = PanelType.WallInternal)
```

File: Environment_Engine/Create/Panel.cs [hunk 2]
```diff
@@ -66,7 +66,7 @@ public static List<Panel> Panels(this BoundaryRepresentation brep, string connec
         [Input("angleTolerance", "The angle tolerance for collapsing to polylines used when generating the external edges of the surfaces")]
         [Input("panelType", "The panel type to assign to the panels, if no input is added roof, floor and wall types are assigned by default")]
         [Output("panel", "An Environment Panels representing a closed space generated from the provided surfaces")]
-        public static List<Panel> Panels(this List<ISurface> surfaces, string connectedSpaceName = null, double angleTolerance = BH.oM.Geometry.Tolerance.Angle, PanelType panelType = PanelType.Undefined)
+        public static List<Panel> Panels(this List<ISurface> surfaces, string connectedSpaceName = null, double angleTolerance = BH.oM.Geometry.Tolerance.Angle, PanelType roofType = PanelType.Roof, PanelType ceilingType = PanelType.Ceiling, PanelType internalFloorType = PanelType.FloorInternal, PanelType externalGrounding = PanelType.SlabOnGrade, PanelType externalWallType = PanelType.WallExternal, PanelType internalWallType = PanelType.WallInternal)
```
Comment (FraserGreenroyd) [inline Environment_Engine/Create/Panel.cs:69]:
```suggestion
        [PreviousVersion("6.3", "BH.Engine.Environment.Create.Panels(System.Collections.Generic.List<BH.oM.Geometry.ISurface>, System.String, System.Double, BH.oM.Environment.Elements.PanelType)")]
        public static List<Panel> Panels(this List<ISurface> surfaces, string connectedSpaceName = null, double angleTolerance = BH.oM.Geometry.Tolerance.Angle, PanelType roofType = PanelType.Roof, PanelType ceilingType = PanelType.Ceiling, PanelType internalFloorType = PanelType.FloorInternal, PanelType externalGrounding = PanelType.SlabOnGrade, PanelType externalWallType = PanelType.WallExternal, PanelType internalWallType = PanelType.WallInternal)
```

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]
Can you also resolve the documentation compliance issues @albinber 😄

**Comment by albinber** [comment]
@bhombot check compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot this is a DevOps instruction. I am requesting neutral checks on: unit-tests

**Review by FraserGreenroyd — APPROVED** [review]
Code reviewed and works well, great job @albinber - not an easy task but a great introduction to our versioning system given the number of methods changed 😄

**Comment by FraserGreenroyd** [comment]
@BHoMBot check ready-to-merge

**Pull request merged**